docker-compose.dev.yml

```diff
@@ -30,6 +30,22 @@ services:
   resource-recorder:
```

Comment:
I think we can add a similar entry for the `auth` service too, with its dependency on the `control-db`.

Reply:
5862f38af4b0c5b5fd28b6d8c51adc3025b59134